apps/cli/src/commands/environment/list.environment.ts

```diff
@@ -24,9 +25,39 @@ export class ListEnvironment extends BaseCommand {
     ]
   }
 
-  async action({ args }: CommandActionData): Promise<void> {
+  getOptions(): CommandOption[] {
+    return [
+      {
+        short: '-p',
+        long: '--page <page>',
+        description: 'Index of the page.'
+      },
+      {
+        short: '-l',
+        long: '--limit <items>',
+        description: 'Total number of items per page.'
+      },
+      {
+        short: '-o',
+        long: '--order <sort order>',
```

Comment:
there is a pattern actually. you need to use `--flag-long-name <type>`. so in your case, it will be `--order <string>`. and it will be the same for the other parameters aswell.

After i merge my newer branch with your PR, you would need to add another field named `choices` which will be an array of accepted string choices for every option

Reply:
Talking about this one. Could you please include the `choices` array for fields that accept a predefined type of value?